src/legacy/core_plugins/metrics/public/components/vis_types/metric/vis.js

```diff
@@ -34,7 +34,8 @@ function getColors(props) {
   if (model.background_color_rules) {
     model.background_color_rules.forEach((rule) => {
       if (rule.operator && rule.value != null) {
-        const value = series[0] && getLastValue(series[0].data) || 0;
+        const value = (series[0] && getLastValue(series[0].data)) ||
```

Comment:
What about to simplify it
`const value = getLastValue((series[0] || series[1] || {}).data || 0);`

But it's optional